src/MeshInertiaCalculator.cc

```diff
@@ -101,12 +107,61 @@ void MeshInertiaCalculator::CalculateMeshCentroid(
   _centreOfMass.SetZ(centroid.Z());
 }
 
+//////////////////////////////////////////////////
+void MeshInertiaCalculator::TransformInertiaMatrixToCOM(
```

Comment:
Since `CalculateMeshCentroid` and `TransformInertiaMatrixToCOM` are unused and untested, I think we should remove them.

Reply:
Removed the unused functions in 7424bab. Would create a PR over at gz-math as soon as possible for the `TransformInertiaMatrixToCOM()` function.